mogp_emulator/Kernel.py

```diff
@@ -725,3 +928,33 @@ def __str__(self):
         :rtype: str
         """
         return "Matern 5/2 Kernel"
+
+class SquaredExponential(SqExpBase,StationaryKernel):
```

Comment:
Yeah, Python does support multiple inheritance for situations like this. I think the rule is the order in which you specify the inheritance determines the priority of where you inherit if both classes implement the same method. I think I avoided any situation where this might occur but might have not thought through fully!